modules/core/04-channel/keeper/upgrade.go

```diff
@@ -257,7 +247,7 @@ func (k Keeper) ChanUpgradeAck(
 		Counterparty:    types.NewCounterparty(portID, channelID),
 		Version:         channel.Version,
 		UpgradeSequence: channel.UpgradeSequence,
-		FlushStatus:     counterpartyFlushStatus, // provided by the relayer
+		FlushStatus:     types.NOTINFLUSH, // TODO: remove flush status from channel end
```

Comment:
temporarily hard code the flush status to the default value so proofs can succeed